server/index.js

```diff
@@ -21,7 +21,7 @@ app.get('/api/comments', (req, res) => {
   });
 });
 
-app.get('/api/reply', (req, res) => {
+app.get('/api/song/:songID/replies', (req, res) => {
```

Comment:
this API is not listed in the README.
It doesn't seem like you need an API for replies... according to your JSON object, replies are also in your API get request for comments.